bin/cordlr

```diff
@@ -2,11 +2,11 @@
 const minimist = require('minimist')
 const Help = require('./methods/help')
 const Version = require('./methods/version')
-const Start = require('./methods/start')
+const Cordlr = require('./methods/start')
```

Comment:
I dont mind **Help**, **Version** and **Start**, It is simple and clean. The reason I changed it is because I did a really bad Copy/Paste from old code I did before you made the update to Master branch.

I will change it to **Start** :)